### ISSUE TITLE
Slide presentation drag gesture is broken with keyboard presented

### ISSUE THREAD

**Issue by PhilipDukhov** [nathantannar4/Transmission#52]
1. While keyboard is presented, drag gesture doesn't work properly
2. Root view(`Color.gray`) has keyboard padding which is not expected (let me know if I should create a separate issue)

![RocketSim_Recording_iPhone_15_Pro_Max_6 7_2024-07-18_4 43 31 PM](https://github.com/user-attachments/assets/813abafc-6288-4e95-b4d9-3366fdeaa5dc)

```swift
struct Issue52: View {
    var body: some View {
        VStack {
            Color.gray.frame(height: 400)
        }
        .presentation(
            transition: .slide(options: .init(options: .init(preferredPresentationBackgroundColor: .clear))),
            isPresented: .constant(true)
        ) {
            TextField("2", text: .constant("1"))
                .frame(maxHeight: .infinity)
                .background(Material.thinMaterial)
        }
    }
}

```

**Comment by PhilipDukhov** [comment]
It seems like fixed on Simulator, but on real device I still encourage issues:


https://github.com/user-attachments/assets/d6c3a509-9fe9-4308-bdd6-7f6c09c6fc7c

**Comment by vpontis** [comment]
@nathantannar4 could we re-open this issue to track this? Or should we make another issue for it? Let us know if there's anything we can do to make this easier to repro or fix.

**Comment by nathantannar4** [comment]
Fixed in `1.4.3`!

**Comment by vpontis** [comment]
Ah this seems to be working great, thank you @nathantannar4!
